src/common/containers/Sets.tsx

```diff
@@ -41,6 +47,12 @@ class SetSummary extends React.Component<{ set: w.Set }> {
 }
 
 class Sets extends React.Component<SetsProps> {
+  public static styles = {
+    newSetButtonLabel: {
+      fontFamily: 'Carter One'
```

Comment:
Let's add a constant like `const WORDBOTS_FONT = 'Carter One';` or something.

Reply:
Let's do that in a separate PR – we use Carter One in a LOT of places ...